### DISCUSSION TITLE
Next Spatial Release?

### DISCUSSION THREAD

**Discussion by b-schmeling** [duckdb/duckdb-spatial#228] [General]
It seems that a new release of DuckDB is on the horizon - there is a 0.9.3 pre-release version and the [docs site](https://duckdb.org/dev/release-dates) mentions a tentative 0.10.0 release on 1/29. 

Will spatial also be released along with it? The nightly version of spatial includes a lot of bug fixes we need, but we would prefer not to pull from the nightly release in our production code. We have a few features on hold right now due to this.

**Comment by Maxxen** [comment]
Yes, we will release the next stable spatial along with the next stable DuckDB release.

**Reply by NathanEmb** [reply]
Any update here? Is it technically released? I went to upgrade my spatial binary that we pulled off the releases tab but it's still at 0.9.1 right now. I keep a copy of the binary in repo so that I don't have to deal with network stuff when I use it.

I tried to find the deployed binary off of the nightly build but it seems it's just on some s3 path somewhere that is obfuscated.

Any tips?

**Reply by NathanEmb** [reply]
Tip: go to actions, find nightly build, scroll all the way down and the downloads for the nightly builds are there.

So I think I'm good I have a binary I can use. An official release version on the GitHub page might be nice for users though

**Reply by dberardo-com** [reply]
are you planning to release also the st_angle function ? https://postgis.net/docs/ST_Angle.html

i mean some sort of function to calculate the relative angle between to vectors, or between a point and the "north line" (azimuth ?)

**Reply by TimJMartin** [reply]
@Maxxen with the announcement yesterday about DuckDB 1.0 do you have an idea on when a new spatial extension release might be?

**Reply by Maxxen** [reply]
@TimJMartin it should already be released! Just "INSTALL spatial; LOAD spatial" from within sql in duckdb v1.0.0 and youll get the latest build.

**Reply by TimJMartin** [reply]
Brilliant, thanks. 

I saw you post a comment about the new VSS extension helping the spatial. Is there going to be a new release with native support for GeoParquet support and some spatial indexing?

**Reply by Maxxen** [reply]
Yeah geoparquet is for sure on the agenda for what im going to work on this cycle. Ive done some prototyping with indexes as well and there shouldnt be any blockers, just a lot of work to do.

**Reply by TimJMartin** [reply]
Sounds great. Look forward to trying it out. Thanks for all your hardwork.
